php/class-wp-cli.php

```diff
@@ -1289,7 +1289,14 @@ public static function get_config( $key = null ) {
 	 * @category Execution
 	 *
 	 * @param string $command WP-CLI command to run, including arguments.
-	 * @param array  $options Configuration options for command execution.
+	 * @param array  $options {
+	 *     Configuration options for command execution.
+	 *
+	 *     @type bool        $launch     Launch a new process, or reuse the existing. Defaults to true.
+	 *     @type bool        $exit_error Exit on error. Defaults to true.
+	 *     @type bool|string $return     Capture and return output, or render in realtime. Defaults to false.
+	 *     @type bool|string $parse      Parse returned output as a particular format. Defaults to false.
```

Comment:
```suggestion
	 *     @type bool|string $parse      Parse returned output as 'json' (string); otherwise, output is unchanged (false). Default: false.
```